apps/dashboard/config/locales/en.yml

```diff
@@ -241,6 +241,7 @@ en:
     jobs_scripts_deleted: "Script successfully deleted!"
     jobs_scripts_submitted: "Successfully submited job %{job_id}."
     jobs_scripts_delete_script_confirmation: "Delete all contents of script?"
+    jobs_scripts_static_field: "Static Value"
```

Comment:
Can we use `Fixed Value` instead here? I think (wonder?) if `static` is a bit esoteric. I think `fixed` may be easier for a layperson to understand.

Reply:
Fixed